react-fela/build.boot

```diff
@@ -0,0 +1,29 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies   '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
```

Comment:
This being React bindings for Fela, I presume this library requires React. `cljsjs/react` and `cljsjs/fela` should be added to `:dependencies`.